docs/webportal/README.md

```diff
@@ -1,134 +1,3 @@
-# Web Portal
+# Webportal of Open PAI
```

Comment:
please update the `zh_CN` doc also

Reply:
add a link to readme, will add a translation in a new pr.